src/linear_algebra/linear_combination.lean

```diff
@@ -3,33 +3,34 @@ Copyright (c) 2017 Johannes Hölzl. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Johannes Hölzl, Mario Carneiro
 
-The module `lc α β` of linear combinations over `β` (`α` is the scalar ring)
+The module `lc ι α β v` of linear combinations over `β`, indexed by ι via v (`α` is the scalar ring)
 -/
 import linear_algebra.basic
 noncomputable theory
 
 open classical function lattice
 local attribute [instance] prop_decidable
 
-variables {α : Type*} {β : Type*} {γ : Type*} {δ : Type*}
-
-/-- The type of linear coefficients, which are simply the finitely supported functions
-from the module `β` to the scalar ring `α`. -/
-@[reducible] def lc (α β) [ring α] [add_comm_group β] [module α β] : Type* := β →₀ α
+/-- The type of linear combinations over an indexed family -/
+@[reducible] def lc (ι : Type*) (α : Type*) [has_zero α] := ι →₀ α
```

Comment:
You are flipping the arguments to `lc`. If we are going to keep this type (which is up for discussion), then I'd really like to have the scalars as first argument.

Reply:
I merged `lc` into `finsupp`.